### ISSUE TITLE
Navigation Drawer not responding.

### ISSUE THREAD

**Issue by Drukey** [IramML/UberClone#14]
Holla IramML! my Spanish isn't really good but my Navigation drawer don't seem to work, the Sign out button, Trip History and Update Information, i am using android Lollipop i guess that's why, also the map on rider home isn't showing.

**Comment by IramML** [comment]
Did you put the realtime database rules to true? Write and read

**Comment by Drukey** [comment]
Yes, i did. The map is now showing. I think it was something with the api key. but the navigation drawer menus aren't responding.
